app/v1/models/caterers.py

```diff
@@ -0,0 +1,25 @@
+from app.v1.models.users import DbUsers
+
+
+class DbCaterers(DbUsers):
```

Comment:
I think this model can better be implemented. The caterer model should only create caterers not users. So with this kind of approach, the data becomes inconsistent. 

I would have expected to see methods like:
- `add_caterer(user_id)`
-  `delete_caterer(user_id)`
-  `get_caterers()`
- `get_caterer(user_id)`

Reply:
Am on it.